com.unity.netcode.gameobjects/Samples~/Bootstrap/.sample.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
and yeah, also rename `Samples~` folder as `Samples` too and check if UPM automagically updates folder name + adds an entry to `package.json` for us when we try to pack the package with UPM (again, as we discussed earlier today).